config/application.rb

```diff
@@ -29,7 +29,7 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
-    config.load_defaults 5.2
+    config.load_defaults 6.0
```

Comment:
looks like safe changes re: what this app does https://guides.rubyonrails.org/configuring.html#default-values-for-target-version-6-0

cc @smudge when you have a chance since i know you've got a deep history of thoughts on 5.2 -> 6.0 -- if you remember anything

Reply:
Taking a look...

I also agree on the webpacker front. 🙂

Reply:
I don't think any of these 6.0 behaviors really apply to this application, or would be things that would make it by without a test failing somewhere.